incubator/tensorflow-inception/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
just fyi - this isn't the right `.helmignore`, you can copy the one from here https://github.com/kubernetes/charts/blob/master/stable/acs-engine-autoscaler/.helmignore but it's not that important